src/webui/www/webui.qrc

```diff
@@ -323,6 +323,7 @@
         <file>private/images/help-about.svg</file>
         <file>private/images/help-contents.svg</file>
         <file>private/images/insert-link.svg</file>
+        <file>private/images/ip-blocked.svg</file>
```

Comment:
What is the source of all these icons? Are they all taken from the qBittorrent GUI?

Reply:
Yes